### PR TITLE
feat(Text): Redlining text component for Teams dark & contrast themes

### PR DESCRIPTION
**Pull request by codepretty**
Updating the colors for Text component for Teams dark & contrast themes.

**Dark theme**
image

**Contrast theme**
image

### CODE REVIEW

File: src/themes/teams-dark/components/Text/textVariables.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+export default (siteVariables): ITextVariables => {
+  return {
+    atMentionOtherTextColor: siteVariables.brand06,
+    atMentionMeTextColor: siteVariables.orange04,
+    disabledTextColor: siteVariables.gray06,
+    errorTextColor: siteVariables.red,
+    successTextColor: siteVariables.green04,
+    timestampTextColor: siteVariables.gray04,
+    timestampHoverTextColor: siteVariables.gray02,
+    importantTextColor: siteVariables.red,
```
Comment (notandrew) [inline src/themes/teams-dark/components/Text/textVariables.ts]: can these either be sorted alphabetically or the same as each other?

File: src/themes/teams-high-contrast/siteVariables.ts [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
+import { white, green } from '../teams/siteVariables'
 
 //
 // COLORS
 //
 export const black = '#000'
 
+export const hyperlinkColor = '#ffff01'
+export const disabledColor = '#3ff23f'
+export const selectionBackgroundColor = '#1aebff'
```
Comment (notandrew) [inline src/themes/teams-high-contrast/siteVariables.ts]: these may be older colors or maybe I have older color redlines... (sending image via teams)

File: src/themes/teams-dark/components/Text/textVariables.ts (hunk 1 above)
Comment (levithomason) [reply src/themes/teams-dark/components/Text/textVariables.ts]: Let's also remove the duplicate `Text` word in each.  Variables are already namespaced by display name:

```jsx
theme.componentVariables.Text().successColor
```

File: src/themes/teams-high-contrast/siteVariables.ts [hunk 3]
```diff
@@ -1,10 +1,17 @@
-import { white } from '../teams/siteVariables'
+import { white, green } from '../teams/siteVariables'
 
 //
 // COLORS
 //
 export const black = '#000'
 
+export const hyperlinkColor = '#ffff01'
+export const disabledColor = '#3ff23f'
```
Comment (levithomason) [inline src/themes/teams-high-contrast/siteVariables.ts]: This feels too generic to be useful in all color contexts.  How can we have a single color for "disabled" without knowing if it is for a foreground, background, border, etc or which component it is for?

Comment (codepretty) [reply src/themes/teams-high-contrast/siteVariables.ts]: I can rename to _accessibleYellow_ and _accessibleGreen_ for now. Although, maybe it should just be a general green and yellow option, like _yellow0x_ and _green0x_...

**Review by hughreeling — APPROVED** [review]

**Review by kuzhelov — APPROVED** [review]

**Pull request merged**